### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by natesulat**
@horationunez Wouldn't mind your review to make sure that the unstoppable Gringo + Google Translate combo didn't produce grammatically correct Spanish

### CODE REVIEW

File: examples/translations/en_es.json [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+                "left_index_full": "Left Index {{finger}}",
+                "left_middle_full": "Left Middle {{finger}}",
+                "left_ring_full": "Left Ring {{finger}}",
+                "left_little_full": "Left Little {{finger}}"
+            },
+            "PII": {
+                "firstName": "First Name",
+                "lastName": "Last Name",
+                "birthDate": "Date of birth",
+                "momsName": "Mother's First Name",
```
Comment (horationunez) [inline examples/translations/en_es.json:47]: this is usually `motherName` and `fatherName`

File: examples/translations/en_es.json [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+                "left_middle_full": "{{finger}} Mediano Izquierdo",
+                "left_ring_full": "{{finger}} Anular Izquierdo",
+                "left_little_full": "{{finger}} Meñique Izquierdo"
+            },
+            "PII": {
+                "firstName": "Primer Nombre",
+                "lastName": "Appellidos",
+                "birthDate": "Año de Nacimiento",
+                "momsName": "Nombre de Madre",
+                "dadsName_full": "Nombre de Padre",
```
Comment (horationunez) [inline examples/translations/en_es.json:267]: I think there is a typo here

**Review by horationunez — CHANGES_REQUESTED** [review]
See my notes on two PII field names. The gringo + google translator esta muy bueno.

File: examples/translations/en_es.json (hunk 1 above)
Comment (natesulat) [reply examples/translations/en_es.json:47]: Sure

File: examples/translations/en_es.json (hunk 2 above)
Comment (natesulat) [reply examples/translations/en_es.json:267]: There is. Fixing...

**Comment by natesulat** [comment]
Vamooo

**Review by horationunez — APPROVED** [review]

**Pull request merged**